core/src/main/java/io/grpc/internal/ManagedChannelImplBuilder.java

```diff
@@ -225,18 +228,20 @@ public int getDefaultPort() {
   public ManagedChannelImplBuilder(String target,
       ClientTransportFactoryBuilder clientTransportFactoryBuilder,
       @Nullable ChannelBuilderDefaultPortProvider channelBuilderDefaultPortProvider) {
-    this(target, null, clientTransportFactoryBuilder, channelBuilderDefaultPortProvider);
+    this(target, null, null, clientTransportFactoryBuilder, channelBuilderDefaultPortProvider);
   }
 
   /**
    * Creates a new managed channel builder with a target string, which can be either a valid {@link
    * io.grpc.NameResolver}-compliant URI, or an authority string. Transport implementors must
    * provide client transport factory builder, and may set custom channel default port provider.
    */
-  public ManagedChannelImplBuilder(String target, @Nullable CallCredentials callCreds,
+  public ManagedChannelImplBuilder(
+      String target, @Nullable ChannelCredentials channelCreds, @Nullable CallCredentials callCreds,
```

Comment:
This needs some javadoc to explain what to pass for `channelCreds`.

Reply:
Added doc, but I'm not sure that is what you mean.